src/flb_config_map.c

```diff
@@ -166,7 +166,7 @@ static int translate_default_value(struct flb_config_map *map, char *val)
         entry->val.s_num = flb_utils_size_to_bytes(val);
     }
     else if (map->type == FLB_CONFIG_MAP_TIME) {
-        entry->val.i_num = flb_utils_time_to_seconds(val);
+        entry->val.s_num = flb_utils_time_to_seconds(val);
```

Comment:
Why do you think this change is necessary when `i_num` is an integer and `flb_utils_time_to_seconds` which you didn't modify anywhere also returns an integer?

Reply:
Please revert this.